pokemon-app/src/styles/MovesList.css

```diff
@@ -244,3 +244,38 @@
     top: 38.4%;
   }
 }
+
+
+/* Styling for effectiveness label */
+.effective-label {
+  box-sizing: border-box;
```

Comment:
I suggest making this global by using   * {box-sizing:border-box} so it applies to all of your elements

Reply:
Thanks Paul. Will definitely take that into account!